src/components/Settings.tsx

```
@@ -475,7 +473,159 @@ const Choice = (props: {
+          }
+          props.ctx.templateTools.splice(props.index, 1);
+          props.ctx.setTemplateTools(structuredClone(props.ctx.templateTools));
+        }}
+      >
+        Delete
+      </Button>
+    </div>
+  );
+};
+
+export default (props: {}) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)